loop/conftest.py

```diff
@@ -1,15 +1,79 @@
-# Configuration file for running our tests
+import ConfigParser
 import pytest
 
+from firefox_puppeteer import Puppeteer
+from marionette_driver.marionette import Marionette
+TIMEOUT = 20
+
 
 def pytest_addoption(parser):
+    """Options to be passed in at run time"""
     parser.addoption(
         "--env",
         action="store",
         help="choose a test environment: staging or production"
     )
+    parser.addoption(
+        "--bin",
+        default='/Applications/FirefoxBeta.app/Contents/MacOS/firefox-bin',
```

Comment:
I know this is prob meant to be Mac only (for now), but in future, you should simply be able to invoke firefox directly on Win/Ubuntu slaves with commands like:
firefox-nighty, firefox-aurora, firefox-release, etc.